spinalcordtoolbox/scripts/sct_image.py

```diff
@@ -169,23 +170,23 @@ def get_parser():
         ],
         required=False)
     orientation.add_argument(
-        '-setorient-data',
-        help='Set orientation of the input image\'s data (modifies ONLY the data array, and leaves the header alone).\n'
-             'Example usage: If `-getorient` returns "RPI", then:\n'
-             '  - Calling `-setorient-data LPI` will flip the LR axis of the data array.\n'
-             '  - Calling `-setorient-data IPR` will rearrange the first and last axes of the data array.\n'
-             '  - In both cases, calling `-getorient` afterwards will still return "RPI", because the header is '
-             'not modified.\n'
-             'WARNING: Use with care, as improper usage may introduce a mismatch between orientation of the header, '
-             'and the orientation of the data array.\n',
-        choices=[
-            'LAS', 'LAI', 'LPS', 'LPI', 'LSA', 'LSP', 'LIA', 'LIP',
-            'RAS', 'RAI', 'RPS', 'RPI', 'RSA', 'RSP', 'RIA', 'RIP',
-            'ALS', 'ALI', 'ARS', 'ARI', 'ASL', 'ASR', 'AIL', 'AIR',
-            'PLS', 'PLI', 'PRS', 'PRI', 'PSL', 'PSR', 'PIL', 'PIR',
-            'SLA', 'SLP', 'SRA', 'SRP', 'SAL', 'SAR', 'SPL', 'SPR',
-            'ILA', 'ILP', 'IRA', 'IRP', 'IAL', 'IAR', 'IPL', 'IPR',
-        ],
+        '-flip',
+        help="Flip an axis of the image's data array. (This will not change the header orientation string.)\n"
+             "(WARNING: This option should only be used to fix the data array when it does not match the orientation "
+             "string in the header. We recommend that you investigate and understand where the mismatch originated "
+             "from in the first place before using this option.)\n"
+             "Example: For an image with 'RPI' in its header, `-flip x` will flip the LR axis of the data array.",
+        choices=['x', 'y', 'z', 't'],
```

Comment:
Could we make `dim_list` a module-level constant, and use it here? After all, the code later in the file assumes that all the choices appear in `dim_list`, so it would be a single source of truth.
```suggestion
        choices=dim_list,
```